vast/basic/mediafile.go

```diff
@@ -1,19 +1,18 @@
-package vast
+package vastbasic
 
-import "github.com/Vungle/vungo/vast/defaults"
+import (
+	"github.com/Vungle/vungo/vast/defaults"
```

Comment:
change back to online import not need the ()

Reply:
Change to one line style.